src/mumble/DirectSound.cpp

```diff
@@ -560,15 +560,15 @@ void DXAudioInput::run() {
 					didsleep = true;
 					firstsleep = false;
 				}
-			} while (static_cast<int>(dwReadyBytes) < sizeof(short) * iFrameSize);
+			} while (static_cast<unsigned int>(dwReadyBytes) < sizeof(short) * iFrameSize);
 
 			// Desynchonized?
 			if (dwReadyBytes > (dwBufferSize / 2)) {
 				qWarning("DXAudioInput: Lost synchronization");
 				dwLastReadPos = dwReadPosition;
 			} else if (bRunning) {
 				if (FAILED(hr = pDSCaptureBuffer->Lock(dwLastReadPos, sizeof(short) * iFrameSize, &aptr1, &nbytes1, &aptr2, &nbytes2, 0))) {
-					qWarning("DXAudioInput: Lock from %ld (%d bytes) failed: hr=0x%08lx",dwLastReadPos, sizeof(short) * iFrameSize, hr);
+					qWarning("DXAudioInput: Lock from %ld (%llu bytes) failed: hr=0x%08lx",dwLastReadPos, sizeof(short) * iFrameSize, hr);
```

Comment:
OTOH, this should be OK if you cast dwLastReadPos to unsigned long long... That an acceptable way to portably print a size_t in my book.

But %zu should work in modern MSVCs (and even not-so modern ones)...

https://github.com/mumble-voip/mumble-releng/blob/master/buildenv/1.3.x/win32-static/patches/test_vsnprintf.c#L103-L104 <- That's a test I created when porting libsndfile from MinGW to MSVC2013 (I believe).

So ust use %zu :)